src/mongocxx/options/find_one_and_replace.hpp

```diff
@@ -19,9 +19,11 @@
 #include <cstdint>
 
 #include <bsoncxx/document/view.hpp>
-#include <mongocxx/options/find_one_and_modify.hpp>
 #include <bsoncxx/stdx/optional.hpp>
```

Comment:
I just checked, and both are necessary to build. The `<bsoncxx/stdx/*>` defines things such as `optional`, and the `<mongocxx/stdx.hpp>` just says to use all of the `bsoncxx::stdx` namespace inside `mongocxx::stdx`.